index.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <link rel="stylesheet" type="text/css" href="./assets/css/home.css">
+</head>
+
+<body>
+<!-- Create a navigation bar with a search engine list -->
+
+<section class="hero is-primary">
+  
+    <div class="hero-body">
+      <p class="title">
+        Studio Ghibli App
+      </p>
+
+      <!-- Main container -->
+<nav class="level">
+
+    <!-- Right side -->
+    <div class="level-right">
+        
+      </div>
+      
+    <!-- Left side -->
+    <div class="level-left">
+      <div class="level-item">
+
+      </div>
+      <div class="level-item">
+        <div class="field has-addons">
+          <p class="control">
+            <input class="input" type="text" placeholder="Find a post">
+          </p>
+          <p class="control">
+            <button class="button">
+              Search
+            </button>
+          </p>
+        </div>
+      </div>
+    </div>
+  
+  </nav>
+
+    </div>
+    
+  </section>
+
+  <div class="container" style="
```

Comment:
What is the purpose of this container? Also, make sure you put styles in the css file.

Reply:
Complete ✅ Thank you.